.github/workflows/ci.yml

```diff
@@ -95,7 +95,12 @@ jobs:
       PYFLUENT_CONTAINER_MOUNT_SOURCE: "/home/ansys/Downloads/ansys_fluent_core_examples"
 
     steps:
-      - uses: actions/checkout@v4
+      - uses: ansys/actions/doc-build@v8
+        with:
+          python-version: ${{ env.MAIN_PYTHON_VERSION }}
+          add-pdf-html-docs-as-assets: true
+          check-links: false
+          needs-quarto: true
```

Comment:
You can not use directly the doc-build action here since PyFluent is using its custom steps for building the HTML files.

Therefore, you need to manually include the steps for:

- Installing Quarto
- Rendering the cheatsheet as PDF
- Copy the rendered PDF to a location within doc/source

Reply:
Pinging @Revathyvenugopal162 so she is aware about this pull-request.

Reply:
Ok. Thank you very much. @jorgepiloto

Reply:
@Revathyvenugopal162 Thank you very much.

Reply:
@jorgepiloto @Revathyvenugopal162 Done. Thank you.